flixel/addons/display/FlxNestedSprite.hx

```diff
@@ -266,14 +266,18 @@ class FlxNestedSprite extends FlxSprite
 					var cos:Float = Math.cos(radians);
 					var sin:Float = Math.sin(radians);
 					
-					var dx:Float = child.relativeX - offset.x;
-					var dy:Float = child.relativeY - offset.y;
+					/* Fix from yannicular */	
```

Comment:
About the `tilesheet rendering`, I'm not sure i got what you mean. Could you give me some more details ? 

On native targets I noticed there is still a glitch on neko, and it works ok with windows. Is there something else I'm missing ?